### DISCUSSION TITLE
Poisson-Laguerre tessellation generation

### DISCUSSION THREAD

**Discussion by itismepeter** [neperfepx/neper#259] [Q&A] [answered]
Dear Romain or fellow Neper users,

I am trying to generate a Laguerre tessellation with randomly placed seeds while specifying the distribution of weights of the seeds or radii (which are equal to the square root of the weights if I understand correctly). As far as I know, this can be achieved by using `-morpho "diameq:"` while generating a tessellation.

However, I was trying to create a tessellation with either uniformly distributed radii or radii from a two-atom distributions, i.e. with only two possible values. I did the latter by specifying a custom distribution in a file:
```
0.05 0.3
0.1 0
0.15 0
0.2 0.7
```
Using this, I expected also the seed weights in the .tess file to attain only two distinct values; however, they do not. Is this the correct approach or am I doing something wrong? Also, is using `diameq:beta(1,1)` a recommended way to obtain uniformly distributed radii (not weights)?

Thank you.
Best regards,
Peter

**Comment by rquey** [comment, marked as answer]
`-morpho` is to specify the cell properties (`diameq`, `sphericity`, etc.). For seeds properties (`coo` and `weight`), use `-morphooptiini`.

>  Also, is using diameq:beta(1,1) a recommended way to obtain uniformly distributed radii (not weights)?

It seems to do the trick, yes.
